shapely/predicates.py

```diff
@@ -532,6 +535,15 @@ def contains(a, b, **kwargs):
     >>> contains(area, None)
     False
     """
+    b = np.asarray(b)
+    if np.issubdtype(b.dtype, np.number):
```

Comment:
This part could be factored out to reduce duplicate code.

And the docstring should mention that you can supply a `(..., 2)` array of numbers which will be interpreted as points.